cantools/database/can/environment_variable.py

```diff
@@ -0,0 +1,161 @@
+# A CAN environment variable.
+
+class EnvironmentVariable(object):
+    """A CAN environment variable.
+
+    """
+
+    def __init__(self,
+                 name,
+                 env_type,
+                 minimum,
+                 maximum,
+                 unit,
+                 initial_value,
+                 env_id,
+                 access_type,
+                 access_node,
+                 comment):
+        self._name = name
+        self._type = env_type
+        self._minimum = minimum
+        self._maximum = maximum
+        self._unit = unit
+        self._initial_value = initial_value
+        self._id = env_id
+        self._access_type = access_type
+        self._access_node = access_node
+        self._comment = comment
+
+    @property
+    def name(self):
+        """The environment variable name as a string.
+
+        """
+
+        return self._name
+
+    @name.setter
+    def name(self, value):
+        self._name = value
+
+    @property
+    def type(self):
```

Comment:
Preferably replace ``type`` and ``id`` with other words to make syntax highlighters happy. Maybe ``kind`` instead of ``type``, or ``env_type``? It's up to you, I don't know what's best.

Reply:
True, I think `env_id` would be the best to deal with `id`, so `env_type` by extension.